docs/wrapper.rst

```diff
@@ -0,0 +1,52 @@
+===============
+Wrapper
+===============
+
+-------------------------------------
+Description
+-------------------------------------
+Note: Wrapper is currently under development. At the moment, its functionality 
+is limited.
+
```

Comment:
```suggestion

.. note::

    Wrapper is currently under development. At the moment, its functionality is limited.

```

Reply:
The wrapper is under development and its functionality is limited.